apps/src/lib/ui/accounts/AddPasswordController.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import AddPasswordForm from './AddPasswordForm';
+
+export default class AddPasswordController {
+  constructor(form, mountPoint) {
```

Comment:
i was having trouble with this case since it isn't a modal. for this form, i need the element to mount in a specific spot (right after the "update" form and before the other optional forms), so i wanted to have an easy way to identify the mount point without having to do even more complicated logic later (i.e., find the update form and append a new element below it to create the mount point for the "add password" form)

do you have a suggestion for that?

Reply:
:man_facepalming: Nope, I read this code thinking it was a modal.  Ignore this comment!

Reply:
haha no problem

Reply:
In that context, does it make sense to pull the hidden form in the haml file up next to the mount point, and only have the one if-statement?

Reply:
yeah, that probably makes more sense. i tend to put hidden elements/forms/etc at the bottom of the file, but since this has logic around it, it's probably better to only have that logic in one place